src/server/saved_objects/client/saved_objects_client.js

```diff
@@ -272,7 +272,6 @@ export class SavedObjectsClient {
       sortField,
       sortOrder,
       fields,
-      includeTypes,
```

Comment:
To prevent breaking the saved objects UI upon merging, can we leave this as an optional parameter and merge it with the `type` parameter? I'll remove it once I update the saved objects UI to stop using it

Reply:
I should've updated all usages of `includeTypes` to `type`